sky/utils/__init__.py

```diff
@@ -1,2 +0,0 @@
-"""Utility functions."""
-from sky.skylet.providers.kubernetes import utils as kubernetes_utils
```

Comment:
One note that I moved this to sky/skylet/__init__.py to avoid a circular import.